ui/desktop/app/templates/scopes/scope/projects/sessions/session.hbs

```diff
@@ -8,9 +8,13 @@
 <Rose::Layout::Page as |page|>
 
   <page.header>
-    <h1 class='session-details-header'>
+    <h1 class='session-details-info'>
       <Hds::IconTile @color='boundary' @icon='terminal' />
       {{@model.displayName}}
```

Comment:
`{{@model.displayName}}` is a text that is not wrapped into any HTML element. Since here we are using the `H1` as a container for: badge + text + time remaining helper I would suggest to wrap `{{@model.displayName}}` within a `span`.

Then within the styles, on line 509: `> * { margin-right: 1rem }` you are givin margin-right to every element, then line 513, 514 and 515 are not need. 

If you have a container `h1` in this case, and elements within the container, wrapping elements makes easier to controll them.

Reply:
Should we be using the new [text](https://helios.hashicorp.design/components/text) HDS components now? I think we mentioned them in our last tech time